base/bindata.go

```diff
@@ -162,7 +162,7 @@ func AssetNames() []string {
 // _bindata is a table, holding each asset generator, mapped to its name.
 var _bindata = map[string]func() (*asset, error){
 	"webui/templates/archived_list.html": webuiTemplatesArchived_listHtml,
-	"webui/static/styles.css": webuiStaticStylesCss,
```

Comment:
This file is autogenerated by go-bindata, and it does not apply go fmt.
So there are two variants:
1) add instructions to README for apply go fmt after generating bindata.go
2) don't apply go fmt for this file